src/main/kotlin/com/atlassian/performance/tools/jiraactions/api/page/WebDriverExtension.kt

```diff
@@ -25,6 +27,19 @@ fun <T> WebDriver.wait(
     ).until(condition)
 }
 
+@JvmOverloads
+fun WebDriver.wait(
+    timeout: Duration,
+    condition: NativeExpectedCondition,
```

Comment:
It appears on the API surface now, so it should be in Gradle `api` scope: https://ecosystem.atlassian.net/browse/JPERF-607